pkg/pods/launch_script_only_test_hoist_launchable/current

```diff
@@ -0,0 +1 @@
+/Users/mpuncel/Development/go/src/github.com/square/p2/pkg/pods/launch_script_only_test_hoist_launchable/installs/testLaunchable
```

Comment:
i'm not sure what those current symlinks are about, i don't remember putting them there, ill make sure tests still pass without them